packages/zoe/src/zoeService/zoeStorageManager.js

```diff
@@ -182,113 +215,207 @@ export const makeZoeStorageManager = (
-      withdrawPayments: escrowStorage.withdrawPayments,
-      initInstanceAdmin,
-      deleteInstanceAdmin,
-      makeInvitation,
-      invitationIssuer,
-      root,
-      adminNode,
-    });
+    const makeInstanceStorageManager = vivifyFarClassKit(
+      instanceBaggage,
+      'InstanceStorageManager',
+      InstanceStorageManagerIKit,
+      initEmpty,
+      {
+        instanceStorageManager: {
+          getTerms() {
+            return instanceRecordManager.getTerms();
+          },
+          getIssuers() {
+            return instanceRecordManager.getIssuers();
+          },
+          getBrands() {
+            return instanceRecordManager.getBrands();
+          },
+          getInstallationForInstance() {
+            return instanceRecordManager.getInstallationForInstance();
+          },
+          async saveIssuer(issuerP, keyword) {
+            const issuerRecord = await issuerStorage.storeIssuer(issuerP);
+            await escrowStorage.createPurse(
+              issuerRecord.issuer,
+              issuerRecord.brand,
+            );
+            instanceRecordManager.addIssuerToInstanceRecord(
+              keyword,
+              issuerRecord,
+            );
+            return issuerRecord;
+          },
+          makeZoeMint(
+            keyword,
+            assetKind = AssetKind.NAT,
+            displayInfo,
+            { elementShape = undefined } = {},
+          ) {
+            // Local indicates one that zoe itself makes from vetted code,
+            // and so can be assumed correct and fresh by zoe.
+            const issuerBaggage = makeScalarBigMapStore('IssuerBaggage', {
+              durable: true,
+            });
+            const localIssuerKit = makeDurableIssuerKit(
+              issuerBaggage,
+              keyword,
+              assetKind,
+              displayInfo,
+              adminNode.terminateWithFailure,
+              { elementShape },
+            );
+            zoeMintBaggageSet.add(issuerBaggage);
+            return wrapIssuerKitWithZoeMint(keyword, localIssuerKit, adminNode);
+          },
+          registerFeeMint(keyword, allegedFeeMintAccess) {
+            const feeIssuerKit = feeMint.getFeeIssuerKit(allegedFeeMintAccess);
+            return wrapIssuerKitWithZoeMint(keyword, feeIssuerKit, adminNode);
+          },
+          getInstanceRecord() {
+            return instanceRecordManager.getInstanceRecord();
+          },
+          getIssuerRecords() {
+            return issuerStorage.getIssuerRecords(
+              // the issuerStorage is a weakStore, so we cannot iterate over
+              // it directly. Additionally, we only want to export the
+              // issuers used in this contract instance specifically, not
+              // all issuers.
+              Object.values(
+                instanceRecordManager.getInstanceRecord().terms.issuers,
+              ),
+            );
+          },
+          initInstanceAdmin(i, instanceAdmin) {
```

Comment:
A lone `i` is not a great variable name for an instance. When I see it my first assumption is that is an integer or possibly some other number.